ui/app/adapters/pki/issuer.js

```diff
@@ -1,7 +1,7 @@
 import ApplicationAdapter from '../application';
 import { encodePath } from 'vault/utils/path-encoding-helpers';
 
-export default class PkiIssuerEngineAdapter extends ApplicationAdapter {
+export default class PkiIssuerAdapter extends ApplicationAdapter {
```

Comment:
Following the pattern we've used elsewhere by keeping the folder name in front of the class name so this is `PkiIssuerAdapter` instead of `IssuerAdapter`